packages/notte-core/src/notte_core/llms/engine.py

```diff
@@ -109,6 +109,10 @@ async def structured_completion(
             if "```json" in content:
                 # extract content from JSON code blocks
                 content = self.sc.extract(content).strip()
+            elif content.startswith('{"json":'):
```

Comment:
can you put {"json": into a class variable and use content = content [len(XYZ):-1] ?